### PR TITLE
Add documentation for Generic channel

### PR DESCRIPTION
**Pull request by Farhanramadhana**

### CODE REVIEW

File: docs/channels/generic.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+## Parameter
+
+To send a message to chatBot you must have some parameters, see the parameter below:
+
+***URL(POST REQUEST)***
+
+ > <https://kanal.kata.ai/receive_message/>webhookId
+
+***REQUEST BODY***
+> {
```
Comment (resir014) [inline docs/channels/generic.md]:
```suggestion
```json
{
```

Example for opening JSON code block. Please do the same for others.

File: docs/channels/generic.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+    userId: "someUserId",
+    messages: [messageObject]
+  }
```
Comment (resir014) [inline docs/channels/generic.md]: I can't use the suggested change feature to fix this, but add the closing ``` to use the code block syntax correctly.

Please do the same for others.

File: docs/channels/generic.md [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+```
+
+## Testing Generic Channel with Postman app
+
+Here, you can use Postman to test send message to chatBot
+![TestingWithPostman](./images/gc/testing_postman.png)
+
+and you will get response from chatBot to your app
+
+![ResponseChannel](./images/gc/response_from_channel.png)
```
Comment (Rafiramadhan) [inline docs/channels/generic.md]: itu id sama ref-id ga di sensor aja?

sama tambahin di akhir kayak
`Now your chatbot is ready`

File: docs/channels/generic.md [hunk 4]
```diff
@@ -0,0 +1,323 @@
+---
+id: generic-channel
+title: Generic Channel
+---
+
+In this documentation, We will learn how to build chatBot app using Kata Platform and connected it to Generic Channel. Generic Channel can be used for the channel that is not yet available in Kata Platform, so you can make a “wrapper”. Here, we use our app that we build using NestJs *(you can use other programming language or framework to make your app)* to communicate with chatBot.
+
+## Get Started
+
+In this tutorial, we gonna create a generic channel using NestJs and connected it with chatbot.
```
[the remark this answers is not in the document]
Comment (Rafiramadhan) [reply docs/channels/generic.md]: atau 
```suggestion
In this tutorial, we are going to create a generic channel using NestJs and connected it with chatbot.
```

**Review by resir014 — APPROVED** [review]

**Pull request merged**